modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -202,6 +224,18 @@ def fetch_appointment_requests
           }
         end
 
+        def ids(ids)
```

Comment:
Yup it was a convenient way of converting them to "id, id, id". I change name as suggested.

Reply:
I looked in that controller and see what you mean. I have no idea why they did that. The typical way would be `ids.join(', ')`. They're working from user input params, so perhaps there's some strange case they're concerned about. Given that we know our inputs are an array of facility ids, I'd suggest we just change to the join method because it's more obvious and no one will have to look up the `row_sep` option to understand what's happening.

Reply:
updated!